<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2a2a2a;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>371</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>286</width>
     <height>371</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 5px;
background-color: #2a2a2a;
color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	background-color: #404040;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 20px;
	background-color: #f73643;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #d63a44;
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="hide">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-bottom: 4px;
	font-size: 20px;
	background-color: #47fc65;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #49cc5f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>5</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
color: white;</string>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="min_age">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>36</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	font-size: 14px;
	color: white;
	background-color: #404040;
	border-radius: 5px;
	padding-left: 7px;
}

QLineEdit::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 14px;
	color: white;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #55ada3;
}</string>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 14px;
	color: white;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #55ada3;
}</string>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Возраст</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>50</y>
      <width>26</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="text">
     <string>до</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="text">
     <string>от</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_age">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>36</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	font-size: 14px;
	color: white;
	background-color: #404040;
	border-radius: 5px;
	padding-left: 7px;
}

QLineEdit::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Таблица</string>
    </property>
   </widget>
   <widget class="QComboBox" name="list_table">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>90</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	border: none;
	border-radius: 5px;
	color: white;
	font-size: 14px;
	padding-left: 5px;
	background-color: #404040;
}

QComboBox::drop-down {
	border:  none;;
	border-radius: 2px;
 }

QComboBox::drop-down::hover {
	background-color: #949494;
 }
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_table">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: #2a2a2a;
	background-color: #fc889e;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #fc7991;
}</string>
    </property>
    <property name="text">
     <string>✘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_table">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 22px;
	color: #2a2a2a;
	background-color: #73ffa1;
	border-radius: 5px;
	font-weight: bold;
}
QPushButton::hover {
	background-color: #54ff8d;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_table">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>140</y>
      <width>186</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	font-size: 14px;
	color: white;
	background-color: #404040;
	border-radius: 5px;
	padding-left: 7px;
}

QLineEdit::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Название таблицы</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
